spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConversionUtils.java

```diff
@@ -27,6 +31,18 @@
  */
 class ConversionUtils {
 
+	/**
+	 * Converters from common types to those used by Spanner.
+	 */
+	public static final Collection<Converter> DEFAULT_SPANNER_WRITE_CONVERTERS = ImmutableSet
```

Comment:
@balopat 

The DefaultConversionService we (and mongo) uses comes with about 90 typical conversions already handled.

However, I also added these here (empty for this PR) for our own conversions (such as from java Date to spanner Date). I want to add those custom conversions and tests for them in the next PR. This PR is just for setting up the classes to use them.

Reply:
sounds good!